docs/topics/feed-exports.rst

```diff
@@ -233,6 +234,13 @@ The serialization format to be used for the feed. See
 
 .. setting:: FEED_EXPORT_FIELDS
 
+FEED_EXPORT_ENCODING
+--------------------
+
+The encoding to be used for the feed. Defaults to UTF-8.
```

Comment:
Let it be so =) I updated the PR with your suggestion

On Tue, Jul 12, 2016 at 3:41 PM, Elias Dorneles <notifications@github.com>
wrote:

> In docs/topics/feed-exports.rst
> <https://github.com/scrapy/scrapy/pull/2034#discussion_r70439572>:
>
> > @@ -233,6 +234,13 @@ The serialization format to be used for the feed. See
> >
> >  .. setting:: FEED_EXPORT_FIELDS
> >
> > +FEED_EXPORT_ENCODING
> > +--------------------
> > +
> > +The encoding to be used for the feed. Defaults to UTF-8.
>
> Concretely, here is my suggestion:
>
> FEED_EXPORT_ENCODING
> --------------------
>
> Default: ``None``
>
> The encoding to be used for the feed.
>
> If unset or set to ``None`` (default) it uses UTF-8 for everything except JSON output,
> which uses safe numeric encoding (``\uXXXX`` sequences) for historic reasons.
>
> Use ``utf-8`` if you want UTF-8 for JSON too.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/scrapy/scrapy/pull/2034/files/df4190bfd442de7b829a06059d744491455f593c#r70439572>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AC-3KEN03iDEZO6Mxvay9Ud84SiwrbUkks5qU5mdgaJpZM4Iu3Yz>
> .
>